docs/tutorial/diffusion-tractography.md

```diff
@@ -3,15 +3,13 @@
-
-The first step of diffusion preprocessing often involves processing the anatomical images. In order to guarantee that any generalizations regarding location made from the preprocessed diffusion data is anatomically-informed, we must have both of our anatomical (T1w or T2w) images and our diffusion MRI images **aligned**. One way we can make this easier for [QSIPrep](https://brainlife.io/app/5dc1c2e57f55b85a93bd3021) is by aligning the anatomical images in such a way that the center of the brain is centered in the image. We refer to this as **ACPC-aligned**, as we are aligning the data to the **anterior commissure-posterior comissure plane**. This is the first step in dMRI preprocessing, and it is typically done with the [FSL Anat (T1w)](https://brainlife.io/app/5e3c87ae9362b7166cf9c7f4) app. Once we've centered our anatomical image, we can move onto diffusion MRI preprocessing.
+The first step of diffusion tractography often involves processing the anatomical images. To track in a biologically-informed manner, and to extract major white matter tracts, we must have both anatomical (T1w or T2w) images and diffusion MRI images **aligned**. One way we can make this easier for [QSIPrep](https://brainlife.io/app/5dc1c2e57f55b85a93bd3021) is by aligning the anatomical images so that the center of the brain is centered in the image. We refer to this as **ACPC-aligned**, as we are aligning the data to the **anterior commissure-posterior comissure plane**. This is the first step in dMRI preprocessing, and it is typically done with the [FSL Anat (T1)](https://brainlife.io/app/5e3c87ae9362b7166cf9c7f4) app. We will then need to generate cortical and white matter surfaces and brain region parcellations using [Freesurfer](https://brainlife.io/app/5fe1056057aacd480f2f8e48). These will be used for segmenting the major white matter tracts following tractography. Once we've processed our anatomical image, we can move on to diffusion MRI preprocessing.
```

Comment:
This paragraph is almost an exact repeat of the paragraph above

Reply:
very true!